[MKC-1921] mblock getting started (#2253)
arduino/docs-content: content/hardware/08.edu/solution-and-kits/alvik/tutorials/getting-started-mblock/getting-started-mblock.md

## What’s Included in the Box 

When unboxing Alvik, you should find the following items:

- Alvik robot
- USB-C® cable (located under the cardboard tray)

## Update Firmware with Alvik Updater

Updating the firmware on your Alvik ensures it has the latest features, bug fixes, and performance improvements. Regular updates help maintain compatibility with the mBlock software and ensure optimal performance of your robot.

![Alvik Firmware Updater](assets/AlvikFirmwareUpdater30sec.gif)

1. Visit https://alvikupdate.arduino.cc.
2. Connect Alvik to your computer.
3. Turn ON your Alvik.
4. Click the "Connect" button.
5. **A pop-up window will appear** prompting you to select the COM port. Choose the correct port and confirm.
6. Once connected, click the "Update" button and wait for the process to complete.

## Preparing Your Environment

During the installation process, you may **need to install additional drivers**, that require access to the **administrator login password** to your OS.

### Supported Systems

You can use a computer with different operating systems like:

> * Windows

Reviewer: we we want a comma in this one too or none to have comma?

> * Mac, 

Reviewer: macOS is the OS for Mac. This is a small and not important detail perhaps but it was what we have been used in some cases like https://docs.arduino.cc/tutorials/alvik/user-manual/

* Chromebook, 
* Linux. 

> ***Alvik is not yet supported by mBlock mobile app for Android or IOS devices.***

Reviewer: ```suggestion
***Alvik is not yet supported by mBlock mobile app for Android or iOS devices.***
```

### MacOS

To install mBlock on macOS:

1. Click on the "Download for Mac" button on the [Download page](https://mblock.cc/pages/downloads), wait for the download to finish, and find the file in your Downloads folder.
2. Double-click the downloaded file to mount the `.dmg` file. Then, drag the Panda icon to the Applications folder.

	![Install on mac](assets/install-mac.png)

3. Open the Applications folder, locate the mBlock app, and open it. The first time you open the software, right-click (or Control-click) on the app and select "Open" from the dropdown menu.

### Chromebook or Linux

If you're using a **Chromebook** ([read the full guide here](https://support.makeblock.com/hc/en-us/articles/19412317319191-Device-Connection-Guide-for-Chromebook-Latest-Version)) or a **Linux** machine, the installation process is different because there is no installable version of the software available. Instead, you'll use the online version of mBlock, which requires a special driver to connect to the robot.

> To find the "mLink - mBlock web version driver" and download the appropriate file for your operating system.

Reviewer: ```suggestion
Here you can find the "mLink - mBlock web version driver" and download the appropriate file for your operating system.
```

**6.** Click the **Connect** button in the pop-up window.

In the connect window, select the **Serial Port**, in Windows you might see something like COM4 (or another number), while on other operating systems, it could be something like `/dev/tty.[yourSerialPortName]`. mBlock typically detects the correct port automatically, but if you encounter any issues, make sure the selected port is the right one.

![mblock COM port selection](assets/COM-port.png)

**7.** You are **ready** to work!

## Blocks 

Explore mBlock to start programming your Alvik. In the center of the interface, you’ll find a list of visual blocks, each representing a command that Alvik can execute. These blocks are grouped **into color-coded categories based on the type of action** they perform.

![Command List](assets/actions.png)When you're creating a sequence of commands for Alvik you **always need to start by stating the initial event**:

![Event in the action menu](assets/events.png)

* `When Alvik starts`  - the code will be executed the moment you turn ON Alvik.

or

* `When button [] is pressed` - the code will be executed the moment you press the designated button.

> ### Blocks Parameters

Reviewer: ```suggestion
### Block Parameters
```

Every parameter inside the **rectangle area**, for example the number `5` or the `cancel ▼` element in the dropdown menu can be edited, to give Alvik a different behavior.

![yellow LED rectangle commands](assets/yellow-blink.png)

> Some blocks, for example in the`Control` section, have an **hexagonal** tile `⬣` parameter, where you can drag and drop in an entire block, allowing you to nest commands like for `if <   > then {   }`.

Reviewer: ```suggestion
Some blocks, for example in the`Control` section, have a **hexagonal** tile `⬣` parameter, where you can drag and drop in an entire block, allowing you to nest commands like for `if <   > then {   }`.
```
